docker/scripts/install_python_version

```diff
@@ -19,6 +19,7 @@ fi
 
 # Download requirements files from GitHub
 wget -nv https://raw.githubusercontent.com/h2oai/h2o-3/${H2O_BRANCH}/h2o-py/test-requirements.txt
+wget -nv https://raw.githubusercontent.com/h2oai/h2o-3/${H2O_BRANCH}/h2o-py/test-requirements-py3.txt
```

Comment:
I think we should rather have test-requirements.txt for Python 3 (which most people use) and add test-requirements-py2.txt for legacy

=> py3 would be the expected default

Reply:
good call